src/libcommonserver/io/iohelper_mac.cpp

```diff
@@ -222,15 +222,14 @@ bool IoHelper::getRights(const SyncPath &path, bool &read, bool &write, bool &ex
     ItemType itemType;
     const bool success = getItemType(path, itemType);
     if (!success) {
-        LOGW_WARN(logger(),
-                  L"Failed to check if the item is a symlink: " << Utility::formatIoError(path, itemType.ioError).c_str());
+        LOGW_WARN(logger(), L"Failed to get item type - " << Utility::formatIoError(path, itemType.ioError).c_str());
```

Comment:
LOGW_WARN(logger(), L"Failed to get item type: " << Utility::formatIoError(path, itemType.ioError).c_str());

Reply:
I updated this message (and similar ones) in commit: https://github.com/Infomaniak/desktop-kDrive/pull/68/commits/a5c903deb7f5fd45adf9baed8c722b029eb1d50f